wgsl/index.bs

```diff
@@ -2987,10 +2987,10 @@ TODO: deduplicate these tables
 ## Texture built-in functions ## {#texture-builtin-functions}
 
 <pre class='def'>
-`vec4<type> textureLoad(texture_ro, i32 coords, i32 level_of_detail)`
-`vec4<type> textureLoad(texture_ro, vec2<i32> coords, i32 level_of_detail)`
-`vec4<type> textureLoad(texture_ro, vec3<i32> coords, i32 level_of_detail)`
-  %32 = OpImageLoad %v4float %texture_ro %coords Lod %level_of_detail
+`vec4<type> textureLoad(texture_ro, i32 coords)`
+`vec4<type> textureLoad(texture_ro, vec2<i32> coords)`
+`vec4<type> textureLoad(texture_ro, vec3<i32> coords)`
```

Comment:
@jdashg to follow up to fix the types here. `texture_ro` doesn't exist, these should be specific overloads with `texture_ro_1d` etc.